kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/status/NetworkStatusServiceAdapter.java

```diff
@@ -210,9 +212,11 @@ private void setIpv4DhcpClientProperties(GwtNetInterfaceConfig gwtConfig,
                     gwtConfig.setIpAddress(firstAddress.getAddress().getHostAddress());
                     gwtConfig.setSubnetMask(NetworkUtil.getNetmaskStringForm(firstAddress.getPrefix()));
                 }
-                if (address.getGateway().isPresent()) {
-                    gwtConfig.setGateway(address.getGateway().get().getHostAddress());
-                }
+
+                String gatewayAddress = address.getGateway().isPresent() ? address.getGateway().get().getHostAddress()
+                        : EMPTY_STRING;
```

Comment:
do we really need a constant for the empty string?

Reply:
Eh... not really. It's just for better readability. It doesn't hurt though...

Reply:
IMHO it's not better readable, one can interpret is as a constant for 'empty' and it feels like:

```java
private static final int ONE = 1;
```

But if you want, we can keep it.

Reply:
I'll keep it